--- src/main/kotlin/BUILD.bazel
+++ src/main/kotlin/BUILD.bazel
@@ -41,6 +41,7 @@
         ":skip-code-gen",
         "//src/main/kotlin/io/bazel/kotlin/compiler",
         "@com_github_jetbrains_kotlin//:lib/kotlin-compiler.jar",
+        "@com_github_jetbrains_kotlin//:lib/jvm-abi-gen.jar",
     ],
     main_class = "io.bazel.kotlin.builder.cmd.Build",
     visibility = ["//visibility:public"],
